customTests/azure_gpu_clock_throttling.nhc

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
```

Comment:
please add a comment referencing this URL for throttle codes: https://docs.nvidia.com/deploy/nvml-api/group__nvmlClocksThrottleReasons.html#group__nvmlClocksThrottleReasons_1gb9f319785224c8cdbcaa1613c43e314e